src/custom/components/Menu/index.tsx

```diff
@@ -254,6 +256,14 @@ export function Menu({ darkMode, toggleDarkMode }: MenuProps) {
             Code
           </span>
         </MenuItem>
+        {account && (
+          <MenuItem id="link" href={getExplorerLink(chainId || 1, account, ExplorerDataType.ORDERS)}>
```

Comment:
I see the confusion here.

This is not the explorer we are looking for :sweat_smile: 

I have what you need in a PR that I'm about to merge https://github.com/gnosis/cowswap/blob/55ce1d0e0544d5ca960f48bf4ed1713ac2dd2143/src/custom/utils/explorer.ts#L43